client/settings/fraud-protection/advanced-settings/cards/ip-address-mismatch.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import FraudProtectionRuleCard from '../rule-card';
+import FraudProtectionRuleDescription from '../rule-description';
+import FraudProtectionRuleToggle from '../rule-toggle';
+
+const IPAddressMismatchRuleCard = () => (
+	<FraudProtectionRuleCard
+		title={ __( 'IP Address Mismatch', 'woocommerce-payments' ) }
+		description={ __(
+			'This filter screens for customer IP (Internet Protocol) addresses that are in a different ' +
+				'country than the customer indicated in their billing address.',
```

Comment:
> This filter screens for customer IP (Internet Protocol) addresses that are in a different country than the customer indicated in their billing address.

This sentence seems unclear to me. A suggestion for clarity might be:

> This filter screens the customer's IP (Internet Protocol) address to see if it is in a different country than indicated in their billing address.

Reply:
Addressed in [6625954](https://github.com/Automattic/woocommerce-payments/pull/5925/commits/66259540f940854e1f5cecc82b921bc7be2d6dfa). Thanks for pointing that out.